packages/deployed-backend-client/src/deployed_backend_client.ts

```diff
@@ -69,6 +69,9 @@ export class DefaultDeployedBackendClient implements DeployedBackendClient {
         .filter((stackSummary: StackSummary) => {
           return stackSummary.StackStatus !== StackStatus.DELETE_COMPLETE;
         })
+        .filter((stackSummary: StackSummary) => {
+          return this.isSandboxStack(stackSummary.StackName);
+        })
         .map(async (stackSummary: StackSummary) => {
           const deploymentType = await this.tryGetDeploymentType(stackSummary);
 
```

Comment:
Do we now even need this call? Isn't it guaranteed that whatever got filtered is `Sandbox` deploymentType by definition?

Reply:
I think the idea is that it's possible for a stack to be named in way that matches the heuristic, but isn't truly a sandbox. The stack metadata is the best way to get the deploymentType.